src/Objs/Author/AuthorEditingConfig.js

```diff
@@ -23,8 +23,16 @@ Scrivito.provideEditingConfig("Author", {
     },
   },
   properties: ["title", "description", "image"],
-  propertiesGroups: [socialCardsPropertiesGroup, metadataPropertiesGroup],
+  propertiesGroups: [
+    {
+      title: "Social cards",
```

Comment:
Please do not inline `title: "Social cards"` but instead keep `socialCardsPropertiesGroup` as they used to be.